Handwriting/HandwritingIntentCSSValue.md

```diff
@@ -97,12 +97,12 @@ All CSS properties have computed values for all elements. The enablement of hand
 1. If the computed value for `touch-action` on `element` and all of its ancestors include either keyword `auto`, `handwriting`, or `manipulation`, **enable handwriting**.
 2. If the computed value for `touch-action` on `element` or any of its ancestors does not include either keyword `auto`, `handwriting`, or `manipulation`, **disable handwriting**.
 
-### Caveats
+### Caveats / Cons
 
 A few pain points have been brought up that are worth discussion:
 * Web pages that currently have the `touch-action` property set for different elements will lose the handwriting capabilities on this element even if they don't want to disable it. When the new keyword ships, the absence of the value will be interpreted as the author of the webpage intentionally disabling handwriting.
 * Authors that specify `touch-action: manipulation` will be enabling `handwriting`, even when they might not want the behavior enabled in their webpage. These authors would then need to update their webpages to explicitly mention which behaviors they want, i.e. : `touch-action: pan-x pan-y pinch-zoom`.
-
+*   Using `touch-action` restricts handwriting implementations to touch controls (such as stylus and touch), even though a platform could support handwriting capabilities for other controls, like mouse pointer events.
```

Comment:
I think it's more appropriate to use `touch input devices` instead of `controls`.